### ISSUE TITLE
Fix timeout exception handling

### ISSUE THREAD

**Issue by davidjiglesias** [wazuh/wazuh#14882]
There are several parts of the API and framework code where we are not properly handling a timeout exception.

In this case, the code looks like:

```python
raise WazuhClusterError(3020) if "timeout" in response else e
```
However, the response can contain the word `Timeout` with caps, instead of `timeout`, raising a different exception.

We should review and fix all code lines with this problem.

**Comment by AGAvila** [comment]
I have been searching in the `framework` and `api` folders in order to try to find the "typos" that can potentially cause a wrong exception to raise. 

I believe that I have found all the strings that are necessary to change. However, I have to take a second look to make sure that I have not left anything behind. Also, further testing is needed to check that the changes have not caused unintended consequences.

Regards,
Álvaro

**Comment by AGAvila** [comment]
Hi team,

It seems that this issue was already fixed in this PR: #15220.

Regards,
Álvaro

**Comment by Selutario** [comment]
This was already fixed in:
- https://github.com/wazuh/wazuh/commit/a185bba110c9681e3bf95a31af8cba5a99ea895f#diff-87fd98b3632496277a1170da01a0cf0c2b03e72b930455c52cf45167321a7ad8
